be/src/exec/json_parser.cpp

```diff
@@ -541,4 +545,9 @@ Status ExpandedJsonArrayParserWithRoot::advance() noexcept {
     return Status::OK();
 }
 
+std::string format_json_parse_error_msg(const std::string& raw_error_msg) {
+    // the keywords "parse error" will be used in FE to determine the error type.
+    return fmt::format("parse error. {}", raw_error_msg);
```

Comment:
Is there no thrift protocal to unify the type info `parse error` or other types?

Reply:
no, currently depends on the keywords